FocusTests/PrismSpec.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  PrismSpec.swift
+//  Focus
+//
+//  Created by Robert Widmann on 7/4/15.
+//  Copyright © 2015 TypeLift. All rights reserved.
+//
+
+import XCTest
+import SwiftCheck
+import Focus
+
+/// For now, until I can think of a sensible way to generate Prisms.
+let prismNilGen = Gen.pure(Prism<Int, Int, String, String>(tryGet: { _ in nil }, inject: { $0.characters.count }))
+
+class PrismSpec : XCTestCase {
```

Comment:
This will stay here until (*fingers crossed*) we get conditional extensions.  No sense asking for an ArrowOf/IsoOf<T, U?> where U? can't ever be Equatable or Hashable.